runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -1707,6 +1707,10 @@ J9::CodeGenerator::doInstructionSelection()
 
    bool fixedUpBlock = false;
 
+   if (self()->comp()->getOption(TR_SplitWarmAndColdBlocks) &&
+       !self()->comp()->compileRelocatableCode())
+      setInstructionSelectionInWarmCodeCache();
```

Comment:
nitpick, but probably worth adding brace brackets here, since the if condition spans multiple lines.

Reply:
Done